libs/remix-lib/src/execution/txExecution.ts

```diff
@@ -104,16 +104,32 @@ export function checkVMError (execResult, abi) {
           if (!sign) continue
           if (returnDataHex === sign.replace('0x', '')) {
             customError = item.name
-            decodedCustomErrorInputs = fn.decodeFunctionData(fn.getFunction(item.name), returnData)
+            const functionDesc = fn.getFunction(item.name)
```

Comment:
Can you add a comment for line 107 to 123 explaining what operation is performed?